aiida/restapi/common/utils.py

```diff
@@ -56,6 +56,12 @@ def default(self, o):
                                    o.hour).zfill(2), str(o.minute).zfill(2),
                                          str(o.second).zfill(2)])
 
+        # To support bytes objects, try to decode to a string
+        try:
+            return o.decode('utf-8')
+        except (UnicodeDecodeError, AttributeError):
+            pass
```

Comment:
I think an `elif` for bytes type makes more sense.

Reply:
This discussion here seems to prefer the duck-typing version: https://stackoverflow.com/questions/34869889/what-is-the-proper-way-to-determine-if-an-object-is-a-bytes-like-object-in-pytho